src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -30,6 +30,20 @@ internal static partial class ExpressionSyntaxExtensions
                 return expression;
             }
 
+            // Throw expressions are not permitted to be parenthesized:
+            //
+            //     "a" ?? throw new ArgumentNullException()
+            //
+            // is legal whereas
+            //
+            //     "a" ?? (throw new ArgumentNullException())
```

Comment:
nice catch.

Reply:
No, it was a throw. (I'm sorry. :smile:)